python_modules/libraries/dagster-dbt/dagster_dbt/dbt_manifest.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+from typing import Any, Mapping, TypeAlias, Union, cast
+
+import dagster._check as check
+import orjson
+
+DbtManifest: TypeAlias = Union[Mapping[str, Any], str, Path]
```

Comment:
We might want to call this something like `DbtManifestParam` to make it clear that it doesn't directly hold the manifest.